**Linux** [Build 8202](http://brontes.lab.eng.brq.redhat.com/viewLog.html?buildId=71420&buildTypeId=WFPR) outcome was **FAILURE** using a merge of beb5af1211bb2eca480adf89c40ca2ccf6e2b354
Summary: Tests failed: 2 (2 new), passed: 3198, ignored: 371 Build time: 00:55:12
### Failed tests
```
org.jboss.as.test.integration.ejb.mdb.ejb2x.MDB20TopicTestCase.testEjb20TopicMDBs: java.lang.AssertionError: Reply message was null on reply queue: ActiveMQQueue[ejb2x/replyQueueA]
	at org.jboss.as.test.integration.ejb.mdb.ejb2x.MDB20TopicTestCase.testEjb20TopicMDBs(MDB20TopicTestCase.java:117)
------- Stdout: -------
 [0m03:54:48,039 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-3) WFLYSRV0028: Stopped deployment mdb.jar (runtime-name: mdb.jar) in 18ms
 [0m [0m03:54:48,044 INFO  [org.jboss.as.repository] (management-handler-thread - 2) WFLYDR0002: Content removed from location /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/basic/target/jbossas/standalone/data/content/ce/987f3aba124d0dfa1d5da08f5168618e48f7d5/content
 [0m [0m03:54:48,044 INFO  [org.jboss.as.server] (management-handler-thread - 2) WFLYSRV0009: Undeployed "mdb.jar" (runtime-name: "mdb.jar")
 [0m03:54:48,053 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "remove",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "ejb2x/queue")
    ]
}
03:54:48,063 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "remove",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "ejb2x/replyQueue")
    ]
}
03:54:48,068 INFO  [org.jboss.as.test.integration.ejb.mdb.ejb2x.AbstractMDB2xTestCase] (main) MDB20TopicTestCase.jar:
/org/
/org/jboss/
/org/jboss/as/
/org/jboss/as/test/
/org/jboss/as/test/integration/
/org/jboss/as/test/integration/ejb/
/org/jboss/as/test/integration/ejb/mdb/
/org/jboss/as/test/integration/ejb/mdb/ejb2x/
/org/jboss/as/test/integration/ejb/mdb/ejb2x/MDB20TopicTestCase$JmsQueueSetup.class
/org/jboss/as/test/integration/ejb/mdb/ejb2x/EJB2xMDB.class
/org/jboss/as/test/integration/ejb/mdb/ejb2x/AbstractMDB2xTestCase.class
/org/jboss/as/test/integration/common/
/org/jboss/as/test/integration/common/jms/
/org/jboss/as/test/integration/common/jms/JMSOperations.class
/org/jboss/as/test/integration/common/jms/JMSOperationsProvider.class
/org/jboss/as/test/integration/common/jms/ActiveMQProviderJMSOperations.class
/org/jboss/as/test/integration/common/jms/JMSOperationsException.class
/org/jboss/as/test/shared/
/org/jboss/as/test/shared/TimeoutUtil.class
/META-INF/
/META-INF/MANIFEST.MF
/META-INF/ejb-jar.xml
/META-INF/jboss-ejb3.xml
 [0m03:54:48,084 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 31) AMQ221003: trying to deploy queue jms.topic.ejb2x/topic
 [0m03:54:48,098 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),


org.jboss.as.test.clustering.messaging.ClusteredMessagingTestCase.testClusteredTopic: java.lang.AssertionError: null
	at org.jboss.as.test.clustering.messaging.ClusteredMessagingTestCase.receiveMessage(ClusteredMessagingTestCase.java:218)
	at org.jboss.as.test.clustering.messaging.ClusteredMessagingTestCase.testClusteredTopic(ClusteredMessagingTestCase.java:169)
------- Stdout: -------
04:18:55,433 WARNING [org.jboss.as.arquillian.container.managed.ManagedDeployableContainer] (main) Bundles path is deprecated and no longer used.
04:18:55,434 INFO  [org.jboss.as.arquillian.container.managed.ManagedDeployableContainer] (main) Starting container with: [/usr/java/latest/bin/java, -D[Standalone], -Xms64m, -Xmx512m, -Xmx512m, -Djava.net.preferIPv4Stack=true, -Djava.net.preferIPv4Stack=false, -Djava.awt.headless=true, -Djboss.modules.system.pkgs=org.jboss.byteman, -Djboss.dist=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/../../../dist/target/wildfly-10.0.0.CR1-SNAPSHOT, -Djboss.dist=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/../../../dist/target/wildfly-10.0.0.CR1-SNAPSHOT, -Djava.net.preferIPv6Addresses=true, -server, -Dts.timeout.factor=100, -Dnode0=::1, -Dnode1=::1, -Dmcast=ff13::1, -Dmcast.ttl=0, -Djbossas.ts.submodule.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering, -Djbossas.ts.integ.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/.., -Djbossas.ts.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/../.., -Djbossas.project.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/../../.., -Djboss.inst=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0, -Djboss.node.name=node-0, -Djboss.messaging.cluster.password=clusterpwd, -ea, -Djboss.home.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0, -Dorg.jboss.boot.log.file=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/standalone/log/server.log, -Dlogging.configuration=file:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/standalone/configuration/logging.properties, -jar, /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/jboss-modules.jar, -mp, /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/modules:/opt/buildAgent/work/a31d203e70e89f90/dist/target/wildfly-10.0.0.CR1-SNAPSHOT/modules:/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/modules, org.jboss.as.standalone, -Djboss.home.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0, -Djboss.server.base.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/standalone, -Djboss.server.log.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/standalone/log, -Djboss.server.config.dir=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/standalone/configuration, -Dts.wildfly.version=10.0.0.CR1-SNAPSHOT, -c=standalone-full-ha.xml]
 [0m04:18:55,830 INFO  [org.jboss.modules] (main) JBoss Modules version 1.4.4.Final
 [0m [0m04:18:56,071 INFO  [org.jboss.msc] (main) JBoss MSC version 1.2.6.Final
 [0m [0m04:18:56,150 INFO  [org.jboss.as] (MSC service thread 1-2) WFLYSRV0049: WildFly Full 10.0.0.CR1-SNAPSHOT (WildFly Core 2.0.0.Beta7) starting
 [0m [0m04:18:57,741 INFO  [org.jboss.as.controller.management-deprecated] (ServerService Thread Pool -- 3) WFLYCTL0028: Attribute 'enabled' in the resource at address '/subsystem=datasources/data-source=ExampleDS' is deprecated, and may be removed in future version. See the attribute description in the output of the read-resource-description operation to learn more about the deprecation.
 [0m [0m04:18:57,934 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0039: Creating http management service using socket-binding (management-http)
 [0m [0m04:18:57,962 INFO  [org.xnio] (MSC service thread 1-1) XNIO version 3.3.1.Final
 [0m [0m04:18:57,969 INFO  [org.xnio.nio] (MSC service thread 1-1) XNIO NIO Implementation Version 3.3.1.Final
 [0m [0m04:18:58,031 INFO  [org.jboss.remoting] (MSC service thread 1-1) JBoss Remoting version 4.0.10.Final
 [0m [0m04:18:58,094 INFO  [org.jboss.as.webservices] (ServerService Thread Pool -- 68) WFLYWS0002: Activating WebServices Extension
 [0m [0m04:18:58,113 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 67) WFLYUT0003: Undertow 1.3.0.Beta13 starting
 [0m [0m04:18:58,117 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-4) WFLYUT0003: Undertow 1.3.0.Beta13 starting
 [0m [33m04:18:58,119 WARN  [org.jboss.as.txn] (ServerService Thread Pool -- 66) WFLYTX0013: Node identifier property is set to the default value. Please make sure it is unique.
 [0m [0m04:18:58,155 INFO  [org.jboss.as.security] (ServerService Thread Pool -- 64) WFLYSEC0002: Activating Security Subsystem
 [0m [0m04:18:58,186 INFO  [org.jboss.as.security] (MSC service thread 1-3) WFLYSEC0001: Current PicketBox version=4.9.3.Final
 [0m [0m04:18:58,229 INFO  [org.jboss.as.jsf] (ServerService Thread Pool -- 52) WFLYJSF0007: Activated the following JSF Implementations: [main]
 [0m [0m04:18:58,249 INFO  [org.jboss.as.naming] (ServerService Thread Pool -- 57) WFLYNAM0001: Activating Naming Subsystem
 [0m [0m04:18:58,273 INFO  [org.jboss.as.clustering.jgroups] (ServerService Thread Pool -- 49) WFLYCLJG0001: Activating JGroups subsystem.
 [0m [0m04:18:58,290 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 44) WFLYCLINF0001: Activating Infinispan subsystem.
 [0m [0m04:18:58,309 INFO  [org.jboss.as.naming] (MSC service thread 1-1) WFLYNAM0003: Starting Naming Service
 [0m [0m04:18:58,390 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 67) WFLYUT0014: Creating file handler for path '/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/welcome-content' with options [directory-listing: 'false', follow-symlink: 'false', case-sensitive: 'true', safe-symlink-paths: '[]']
 [0m [0m04:18:58,403 INFO  [org.wildfly.extension.io] (ServerService Thread Pool -- 43) WFLYIO001: Worker 'default' has auto-configured to 4 core threads with 32 task threads based on your 2 available processors
 [0m [0m04:18:58,420 INFO  [org.wildfly.iiop.openjdk] (ServerService Thread Pool -- 45) WFLYIIOP0001: Activating IIOP Subsystem
 [0m [0m04:18:58,534 INFO  [org.jboss.as.connector.subsystems.datasources] (ServerService Thread Pool -- 39) WFLYJCA0004: Deploying JDBC-compliant driver class org.h2.Driver (version 1.3)
 [0m [0m04:18:58,813 INFO  [org.jboss.as.connector] (MSC service thread 1-3) WFLYJCA0009: Starting JCA Subsystem (IronJacamar 1.2.6.Final)
 [0m [0m04:18:58,887 INFO  [org.jboss.as.ejb3] (MSC service thread 1-1) WFLYEJB0481: Strict pool slsb-strict-max-pool is using a max instance size of 32 (per class), which is derived from thread worker pool sizing.
 [0m [0m04:18:58,888 INFO  [org.jboss.as.ejb3] (MSC service thread 1-1) WFLYEJB0482: Strict pool mdb-strict-max-pool is using a max instance size of 8 (per class), which is derived from the number of CPUs on this host.
 [0m [0m04:18:58,889 INFO  [org.jboss.as.mail.extension] (MSC service thread 1-1) WFLYMAIL0001: Bound mail session [java:jboss/mail/Default]
 [0m [0m04:18:58,889 INFO  [org.jboss.as.connector.deployers.jdbc] (MSC service thread 1-1) WFLYJCA0018: Started Driver service with driver-name = h2
 [0m [0m04:18:58,976 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-2) WFLYUT0012: Started server default-server.
 [0m [0m04:18:59,013 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-1) WFLYUT0018: Host default-host starting
 [0m [0m04:18:59,107 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-2) WFLYUT0006: Undertow HTTP listener default listening on [::1]:8080
 [0m [0m04:18:59,121 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-3) WFLYUT0006: Undertow AJP listener ajp listening on [::1]:8009
 [0m [0m04:18:59,175 INFO  [org.jboss.modcluster] (ServerService Thread Pool -- 70) MODCLUSTER000001: Initializing mod_cluster version 1.3.1.Final
 [0m [0m04:18:59,200 INFO  [org.jboss.modcluster] (ServerService Thread Pool -- 70) MODCLUSTER000032: Listening to proxy advertisements on /ff13:0:0:0:0:0:0:1:23364
 [0m [0m04:18:59,318 INFO  [org.jboss.as.connector.subsystems.datasources] (MSC service thread 1-1) WFLYJCA0001: Bound data source [java:jboss/datasources/ExampleDS]
 [0m [0m04:18:59,382 INFO  [org.wildfly.extension.messaging-activemq] (MSC service thread 1-2) WFLYMSGAMQ0075: AIO wasn't located on this platform, it will fall back to using pure Java NIO. Your platform is Linux, install LibAIO to enable the AIO journal and achieve optimal performance.
 [0m [0m04:18:59,597 INFO  [org.wildfly.iiop.openjdk] (MSC service thread 1-4) WFLYIIOP0009: CORBA ORB Service started
 [0m [0m04:18:59,743 INFO  [org.jboss.ws.common.management] (MSC service thread 1-3) JBWS022052: Starting JBoss Web Services - Stack CXF Server 5.1.0.Final
 [0m [0m04:18:59,766 INFO  [org.jboss.as.server.deployment.scanner] (MSC service thread 1-4) WFLYDS0013: Started FileSystemDeploymentService for directory /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/standalone/deployments
 [0m [0m04:18:59,841 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221000: live Message Broker is starting with configuration Broker Configuration (clustered=true,journalDirectory=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/standalone/data/activemq/journal,bindingsDirectory=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/standalone/data/activemq/bindings,largeMessagesDirectory=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/standalone/data/activemq/largemessages,pagingDirectory=/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/clustering/target/wildfly-clustered-messaging-0/standalone/data/activemq/paging)
 [0m [0m04:18:59,887 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221013: Using NIO Journal
 [0m [0m04:19:00,044 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221043: Protocol module found: [artemis-server]. Adding protocol support for: CORE
 [0m [0m04:19:00,066 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221043: Protocol module found: [artemis-amqp-protocol]. Adding protocol support for: AMQP
 [0m [0m04:19:00,080 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221043: Protocol module found: [artemis-hornetq-protocol]. Adding protocol support for: HORNETQ
 [0m [0m04:19:00,082 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221043: Protocol module found: [artemis-stomp-protocol]. Adding protocol support for: STOMP


```

--- a/testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/parse/ParseAndMarshalModelsTestCase.java
+++ b/testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/parse/ParseAndMarshalModelsTestCase.java
@@ -24,78 +24,27 @@ package org.jboss.as.test.manualmode.parse;
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.PROFILE;
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUBSYSTEM;
 
-import java.io.ByteArrayInputStream;
 import java.io.File;
 import java.io.FileNotFoundException;
-import java.io.InputStream;
 
-import org.jboss.arquillian.container.test.api.ContainerController;
-import org.jboss.arquillian.container.test.api.Deployer;
-import org.jboss.arquillian.container.test.api.Deployment;
-import org.jboss.arquillian.container.test.api.RunAsClient;
-import org.jboss.arquillian.container.test.api.TargetsContainer;
-import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.arquillian.junit.InSequence;
-import org.jboss.arquillian.test.api.ArquillianResource;
 import org.jboss.as.test.shared.FileUtils;
 import org.jboss.as.test.shared.ModelParserUtils;
 import org.jboss.dmr.ModelNode;
 import org.jboss.dmr.Property;
-import org.jboss.shrinkwrap.api.Archive;
-import org.jboss.shrinkwrap.api.ShrinkWrap;
-import org.jboss.shrinkwrap.api.asset.Asset;
-import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.junit.Assert;
-import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.runner.RunWith;
 
 /**
  * Tests the ability to parse the config files we ship or have shipped in the past, as well as the ability
  * to marshal them back to xml in a manner such that reparsing them produces a consistent in-memory configuration model.
- * <p/>
- * <b>Note:</b>The standard {@code build/src/main/resources/standalone/configuration} and
- * {@code build/src/main/resources/domain/configuration} files are tested in the smoke integration module ParseAndMarshalModelsTestCase.
  *
  * @author <a href="kabir.khan@jboss.com">Kabir Khan</a>
  * @author <a href="mailto:darran.lofthouse@jboss.com">Darran Lofthouse</a>
  * @author Brian Stansberry (c) 2011 Red Hat Inc.
  */
-@RunWith(Arquillian.class)
-@Ignore("WFLY-5138")
 public class ParseAndMarshalModelsTestCase {
 
-    @Deployment(name = "test", managed = false, testable = true)
-    public static Archive<?> getDeployment() {
-
-        JavaArchive archive = ShrinkWrap.create(JavaArchive.class, "bogus.jar");
-        archive.addPackage(ParseAndMarshalModelsTestCase.class.getPackage());
-        archive.addClass(FileUtils.class);
-        archive.addClass(ModelParserUtils.class);
-        archive.add(new Asset() {
-            @Override
-            public InputStream openStream() {
-                return new ByteArrayInputStream("Dependencies: org.jboss.staxmapper,org.jboss.as.controller,org.jboss.as.deployment-repository,org.jboss.as.server,org.jboss.as.host-controller,org.jboss.as.domain-management,org.jboss.as.security\n\n".getBytes());
-            }
-        }, "META-INF/MANIFEST.MF");
-        return archive;
-    }
-
-    @Test
-    @InSequence(-1)
-    @RunAsClient
-    public void start(@ArquillianResource ContainerController cc, @ArquillianResource Deployer deployer) {
-        cc.start("default-jbossas");
-        deployer.deploy("test");
-    }
-
-    @Test
-    @InSequence(1)
-    @RunAsClient
-    public void stop(@ArquillianResource ContainerController cc, @ArquillianResource Deployer deployer) {
-        deployer.undeploy("test");
-        cc.stop("default-jbossas");
-    }
+    private static final File JBOSS_HOME = new File(".." + File.separatorChar + "jbossas-parse-marshal");
 
     @Test
     public void testStandaloneXml() throws Exception {
@@ -124,22 +73,22 @@ public class ParseAndMarshalModelsTestCase {
 
     @Test
     public void testStandalonePicketLinkXml() throws Exception {
-        standaloneXmlTest(getGeneratedExampleConfigFile("standalone-picketlink.xml"));
+        standaloneXmlTest(getDocsExampleConfigFile("standalone-picketlink.xml"));
     }
 
     @Test
     public void testStandaloneXtsXml() throws Exception {
-        standaloneXmlTest(getGeneratedExampleConfigFile("standalone-xts.xml"));
+        standaloneXmlTest(getDocsExampleConfigFile("standalone-xts.xml"));
     }
 
     @Test
     public void testStandaloneJtsXml() throws Exception {
-        standaloneXmlTest(getGeneratedExampleConfigFile("standalone-jts.xml"));
+        standaloneXmlTest(getDocsExampleConfigFile("standalone-jts.xml"));
     }
 
     @Test
     public void testStandaloneGenericJMSXml() throws Exception {
-        standaloneXmlTest(getGeneratedExampleConfigFile("standalone-genericjms.xml"));
+        standaloneXmlTest(getDocsExampleConfigFile("standalone-genericjms.xml"));
     }
 
     @Test
@@ -373,11 +322,7 @@ public class ParseAndMarshalModelsTestCase {
     }
 
     private ModelNode standaloneXmlTest(File original) throws Exception {
-        File file = new File("target/standalone-copy.xml");
-        if (file.exists()) {
-            file.delete();
-        }
-        return ModelParserUtils.standaloneXmlTest(original, file);
+        return ModelParserUtils.standaloneXmlTest(original, JBOSS_HOME);
     }
 
     @Test
@@ -421,74 +366,58 @@ public class ParseAndMarshalModelsTestCase {
     }
 
     private void hostXmlTest(final File original) throws Exception {
-        File file = new File("target/host-copy.xml");
-        if (file.exists()) {
-            file.delete();
-        }
-        ModelParserUtils.hostXmlTest(original, file);
+        ModelParserUtils.hostXmlTest(original, JBOSS_HOME);
     }
 
     @Test
-    @TargetsContainer("class-jbossas")
     public void testDomainXml() throws Exception {
         domainXmlTest(getOriginalDomainXml("domain.xml"));
     }
 
     @Test
-    @TargetsContainer("class-jbossas")
     public void test713DomainXml() throws Exception {
         ModelNode model = domainXmlTest(getLegacyConfigFile("domain", "7-1-3.xml"));
         validateJsfProfiles(model);
     }
 
     @Test
-    @TargetsContainer("class-jbossas")
     public void test720DomainXml() throws Exception {
         ModelNode model = domainXmlTest(getLegacyConfigFile("domain", "7-2-0.xml"));
         validateJsfProfiles(model);
     }
 
     @Test
-    @TargetsContainer("class-jbossas")
     public void testEAP600DomainXml() throws Exception {
         ModelNode model = domainXmlTest(getLegacyConfigFile("domain", "eap-6-0-0.xml"));
         validateJsfProfiles(model);
     }
 
     @Test
-    @TargetsContainer("class-jbossas")
     public void testEAP610DomainXml() throws Exception {
         ModelNode model = domainXmlTest(getLegacyConfigFile("domain", "eap-6-1-0.xml"));
         validateJsfProfiles(model);
     }
 
     @Test
-    @TargetsContainer("class-jbossas")
     public void testEAP620DomainXml() throws Exception {
         ModelNode model = domainXmlTest(getLegacyConfigFile("domain", "eap-6-2-0.xml"));
         validateJsfProfiles(model);
     }
 
     @Test
-    @TargetsContainer("class-jbossas")
     public void testEAP630DomainXml() throws Exception {
         ModelNode model = domainXmlTest(getLegacyConfigFile("domain", "eap-6-3-0.xml"));
         validateJsfProfiles(model);
     }
 
     @Test
-    @TargetsContainer("class-jbossas")
     public void testEAP640DomainXml() throws Exception {
         ModelNode model = domainXmlTest(getLegacyConfigFile("domain", "eap-6-4-0.xml"));
         validateJsfProfiles(model);
     }
 
     private ModelNode domainXmlTest(final File original) throws Exception {
-        File file = new File("target/domain-copy.xml");
-        if (file.exists()) {
-            file.delete();
-        }
-        return ModelParserUtils.domainXmlTest(original, file);
+        return ModelParserUtils.domainXmlTest(original, JBOSS_HOME);
     }
 
     private static void validateJsfProfiles(ModelNode model) {
@@ -506,7 +435,7 @@ public class ParseAndMarshalModelsTestCase {
 
     private File getOriginalStandaloneXml(String profile) throws FileNotFoundException {
         return FileUtils.getFileOrCheckParentsIfNotFound(
-                System.getProperty("jboss.inst", "../../.."),
+                System.getProperty("jboss.dist"),
                 "standalone/configuration/" + profile
         );
     }
@@ -538,14 +467,7 @@ public class ParseAndMarshalModelsTestCase {
 
     private File getDocsExampleConfigFile(String name) throws FileNotFoundException {
         return FileUtils.getFileOrCheckParentsIfNotFound(
-                System.getProperty("jboss.inst", "../../.."),
-                "docs/examples/configs" + File.separator + name
-        );
-    }
-
-    private File getGeneratedExampleConfigFile(String name) throws FileNotFoundException {
-        return FileUtils.getFileOrCheckParentsIfNotFound(
-                System.getProperty("jboss.inst", "../../.."),
+                System.getProperty("jboss.dist"),
                 "docs/examples/configs" + File.separator + name
         );
     }
@@ -554,14 +476,14 @@ public class ParseAndMarshalModelsTestCase {
         //Get the standalone.xml from the build/src directory, since the one in the
         //built server could have changed during running of tests
         return FileUtils.getFileOrCheckParentsIfNotFound(
-                System.getProperty("jboss.inst", "../../.."),
+                System.getProperty("jboss.dist"),
                 "domain/configuration"
         );
     }
 
     private File getDomainConfigDir() throws FileNotFoundException {
         return FileUtils.getFileOrCheckParentsIfNotFound(
-                System.getProperty("jboss.inst", "../../.."),
+                System.getProperty("jboss.dist"),
                 "domain/configuration"
         );
     }

--- a/testsuite/integration/src/test/scripts/manualmode-build.xml
+++ b/testsuite/integration/src/test/scripts/manualmode-build.xml
@@ -58,6 +58,14 @@
             <fileset dir="target/jbossas"/>
         </copy>
 
+        <echo message="Copying and configuring instance jbossas-parse-marshal"/>
+        <copy todir="target/jbossas-parse-marshal">
+            <fileset dir="target/jbossas"/>
+        </copy>
+        <copy todir="target/jbossas-parse-marshal/modules">
+            <fileset dir="${jboss.dist}/modules"/>
+        </copy>
+
         <!-- Connects back to original host (node0) -->
         <ts.config-as.add-remote-outbound-connection name="jbossas-with-remote-outbound-connection" node="${node0}"
                                                      remotePort="8080" protocol="http-remoting" securityRealm="PasswordRealm" userName="user1"/>

